bundles/ubuntu20.04/packages.txt.gotmpl

```diff
@@ -4,7 +4,7 @@ nfs-common
 python3-cryptography
 python3-pip
 libseccomp2
-kubectl={{ .KubernetesVersion }}-1.1
-kubelet={{ .KubernetesVersion }}-1.1
-kubeadm={{ .KubernetesVersion }}-1.1
+kubectl={{ .KubernetesVersion }}-2.1
```

Comment:
?

Reply:
Upstream package is `v1.29.4-2.1` https://github.com/mesosphere/konvoy-image-builder/actions/runs/8851323182/job/24382905402?pr=1067#step:10:2540 

Made the same change in `kubernetes_deb`variable.